vmaas_sync/advisory_sync.go

```diff
@@ -229,7 +237,7 @@ func downloadAndProcessErratasPage(iPage int, modifiedSince *string) (*vmaas.Err
 	return errataResponse, nil
 }
 
-func vmaasErrataRequest(iPage int, modifiedSince *string) (*vmaas.ErrataResponse, error) {
+func vmaasErrataRequest(iPage int, modifiedSince *string, syncCheck bool) (*vmaas.ErrataResponse, error) {
```

Comment:
Use `pageSize int` argument instead of `syncCheck bool` and you can omit `if syncCheck` condition. Moreover the method head will be more understandable. Just use `advisoryPageSize` when used for standard sync and `1` when used for count check. Btw., can not we use page size 0 here?

Reply:
When I tried `pageSize` 0, request timed out.